packages/embedded/aot-native.properties

```diff
@@ -1,23 +0,0 @@
-#
-# Copyright (c) 2024 Elide Technologies, Inc.
-#
-# Licensed under the MIT license (the "License"); you may not use this file except in compliance
-# with the License. You may obtain a copy of the License at
-#
-# https://opensource.org/license/mit/
-#
-# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
-# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations under the License.
-#
-
-graalvm.config.enabled = true
-known.missing.types.enabled = true
-logback.xml.to.java.enabled = false
```

Comment:
these are being dropped bc they are unused? or, what's the reason?

Reply:
they are currently unused, aot plugin is not being applied yet.